bin/arg.mli

```diff
@@ -7,6 +7,14 @@ include module type of struct
 end
 
 module Path : sig
+  module External : sig
```

Comment:
Could you clarify why we need separate `Path.External` modules here, in addition to `Path.External` from Stdune? It's a bit confusing.

Reply:
(I would expect to see some validation code added somewhere, to catch the garbage, but I don't see it.)